packages/dev-server/README.md

```diff
@@ -28,6 +28,13 @@ module.exports = createConfig([
 ```
 
 
+## Pitfalls
+
+Make sure you use this `dev-server` block after your `entryPoint()` blocks. The dev server will extend all of them with the `'webpack/hot/only-dev-server'` entry point by default.
```

Comment:
This is essentially false. Your dev-server can be after or before your main entry point. It has no impact what so ever. I used to put them at the beginning and it made no difference.